.github/workflows/test.yml

```diff
@@ -189,3 +189,23 @@ jobs:
         run: npm run test:local:node:lokijs
       - name: Stop Consumer API
         run: docker compose -f ./backbone/docker-compose/docker-compose.test.yml down
+  integration-test:
+    name: Run Integration Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install script dependencies
+        run: npm install --prefix ./.ci
+      - name: Setup dotnet
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: "7.0.x"
+      - name: Build project
+        run: dotnet restore "Backbone.sln" && dotnet build  /property:WarningLevel=0 --no-restore "Backbone.sln"
+      - name: Run Consumer API and Admin Ui/API
+        run: docker compose -f ./docker-compose/docker-compose.test.yml up consumer-api admin-ui seed-mssql seed-client -d
+      - name: Run tests
+        run: dotnet test --no-restore --no-build "Backbone.sln" --filter "Category=Integration&TestCategory!~ignore"
+      - name: Stop Consumer API
+        run: docker compose -f ./docker-compose/docker-compose.test.yml down
```

Comment:
Extract these commands into a zx script, so that everyone can run the tests manually on their PC.

Same for the Consumer API commands

Reply:
> Same for the Consumer API commands

Are you referring to the transport tests?

Reply:
Oops. Yes, of course I do :)